README.md

```diff
@@ -935,6 +935,14 @@ your application.
     # more settings
   }
   ```
+  
+* <a name="letter-opener"></a>
```

Comment:
Newer version of Rails have built-in Emacs preview functionality, so I wouldn't recommend this.